README.md

```diff
@@ -193,6 +193,15 @@ docker run --net=host --rm -it \
 
 And then you can access your backend by vising https://${PROJECT_ID}.appspot.com
 
+#### Dockerfile
+
+You can find the Dockerfile (Debian based) under the `agent` folder. `Dockerfile` build a Go binary 
```

Comment:
This wording isn't clear to me at all.

I would start with more background... e.g.

```
There is a `Dockerfile` in the `agent` directory that can be used to build a Docker image for the agent.

That Docker image contains the agent binary along with the following third-party packages installed via `apt-get`

...
```

Reply:
ack